### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1032] [Galaxy Trivia] [answered]
🟩🟨⭐🟥
🟦🌙☀️🇺🇸
⬜🤐🟥🟥

**Comment by peterbaccaro** [comment, marked as answer]
1. SENEGAL
2. MALAYSIA
3. BAHRAIN

![image](https://github.com/ParanoidUser/yolo/assets/74666079/a1869574-fb93-459a-8127-354b0f8f9ff3)

**Comment by thuongtruong1009** [comment]
Based on the emojis provided, the country is likely the United States of America (USA).

🟩🟨⭐🟥 - The green, yellow, star, and red emojis may represent the flag of the USA.
🟦🌙☀️🇺🇸 - The blue, moon, sun, and USA flag emojis further suggest the United States.
⬜🤐🟥🟥 - The white, zipper-mouth, and two red squares emojis may symbolize the idea of keeping quiet or remaining silent.

Putting it all together, the sequence of emojis seems to represent the United States of America and the concept of silence or secrecy.

**Comment by Ben74x** [comment]
Senegal
Malaysia
Malta
